stdlib/public/core/Integers.swift.gyb

```diff
@@ -1918,6 +1924,18 @@ extension BinaryInteger {
 }
 #endif
 
+extension BinaryInteger {
+  // FIXME(integers): inefficient. Should get rid of _word(at:) and
+  // countRepresentedWords, and make `words` the basic operation.
```

Comment:
Do you want to use this to speed it up?

https://gist.github.com/dabrahams/976827a900b22689b09e14e7ffeecdcd